code/modules/mob/living/silicon/pai/emote.dm

```diff
@@ -0,0 +1,163 @@
+/mob/living/silicon/pai/emote(var/act,var/m_type=1,var/message = null, var/auto)
+	if(timestopped)
+		return //under effects of time magick
+	var/param = null
+	if (findtext(act, "-", 1, null))
+		var/t1 = findtext(act, "-", 1, null)
+		param = copytext(act, t1 + 1, length(act) + 1)
+		act = copytext(act, 1, t1)
+
+	if(findtext(act,"s",-1) && !findtext(act,"_",-2))//Removes ending s's unless they are prefixed with a '_'
+		act = copytext(act,1,length(act))
+
+	switch(act)
+		if ("me")
+			if (src.client)
+				if(client.prefs.muted & MUTE_IC)
+					to_chat(src, "You cannot send IC messages (muted).")
+					return
+				if (src.client.handle_spam_prevention(message,MUTE_IC))
+					return
+			if (stat)
+				return
+			if(!(message))
+				return
+			else
+				return custom_emote(1, message)
+
+		if ("custom")
+			return custom_emote(m_type, message)
+
+		if ("deathgasp")
+			message = "<B>[src]</B> shudders violently for a moment, then becomes motionless, its eyes slowly darkening."
+			m_type = VISIBLE
+
+		if ("glare")
+			var/M = null
+			if (param)
+				for (var/mob/A in view(null, null))
```

Comment:
~~AHA SO YOU'RE COPY PASTING?!~~

I guess just remove the arguments to the `view()` entirely.